src/components/atoms/Title/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './style.scss';
+
+const Title = ( { title, titleVariant } ) => {
+    return(
+        <p className={titleVariant ? `title-${titleVariant}` : "title"} >
```

Comment:
why title is `<p>` and not a `<h1>`  or `<h2>`?

Reply:
changed to h1